bin/removeNPMAbsolutePaths

```diff
@@ -16,9 +16,23 @@ const opts = {
   force: false,
 };
 
+let inFields = false;
+
 args.slice(1).forEach((arg) => {
```

Comment:
I'd rather something here that makes easier to add new properties when needed. Something like:
```
for (let i = 0; i < args.length; i++) {
  const arg = args[i];
  switch(expression) {
    case '--force':
      opts.force = true;
      break;
    case '--fields':
      opts.fields = opts.fields || [];
      while(args[i + 1].slice(0, 2) !== '--') {
        opts.fields.push(args[i++]);
      }
      break;
  }
}
```
So each property is 100% handled in its own `case`